[2025-04-01T14:50:21.640+0000] {local_task_job_runner.py:123} INFO - ::group::Pre task execution logs
[2025-04-01T14:50:21.782+0000] {taskinstance.py:2614} INFO - Dependencies all met for dep_context=non-requeueable deps ti=<TaskInstance: my_dag_name.create_table scheduled__2021-12-09T00:00:00+00:00 [queued]>
[2025-04-01T14:50:21.860+0000] {taskinstance.py:2614} INFO - Dependencies all met for dep_context=requeueable deps ti=<TaskInstance: my_dag_name.create_table scheduled__2021-12-09T00:00:00+00:00 [queued]>
[2025-04-01T14:50:21.861+0000] {taskinstance.py:2867} INFO - Starting attempt 1 of 1
[2025-04-01T14:50:22.123+0000] {taskinstance.py:2890} INFO - Executing <Task(BigQueryCreateTableOperator): create_table> on 2021-12-09 00:00:00+00:00
[2025-04-01T14:50:22.220+0000] {standard_task_runner.py:104} INFO - Running: ['***', 'tasks', 'run', 'my_dag_name', 'create_table', 'scheduled__2021-12-09T00:00:00+00:00', '--job-id', '692', '--raw', '--subdir', 'DAGS_FOLDER/con.py', '--cfg-path', '/tmp/tmp1ymdgedn']
[2025-04-01T14:50:22.230+0000] {standard_task_runner.py:105} INFO - Job 692: Subtask create_table
[2025-04-01T14:50:22.223+0000] {warnings.py:112} WARNING - /home/***/.local/lib/python3.12/site-packages/***/task/task_runner/standard_task_runner.py:70: DeprecationWarning: This process (pid=3119) is multi-threaded, use of fork() may lead to deadlocks in the child.
  pid = os.fork()

[2025-04-01T14:50:22.247+0000] {standard_task_runner.py:72} INFO - Started process 3144 to run task
[2025-04-01T14:50:22.500+0000] {task_command.py:467} INFO - Running <TaskInstance: my_dag_name.create_table scheduled__2021-12-09T00:00:00+00:00 [running]> on host 900d630d7d5f
[2025-04-01T14:50:23.808+0000] {taskinstance.py:3134} INFO - Exporting env vars: AIRFLOW_CTX_DAG_OWNER='***' AIRFLOW_CTX_DAG_ID='my_dag_name' AIRFLOW_CTX_TASK_ID='create_table' AIRFLOW_CTX_EXECUTION_DATE='2021-12-09T00:00:00+00:00' AIRFLOW_CTX_TRY_NUMBER='1' AIRFLOW_CTX_DAG_RUN_ID='scheduled__2021-12-09T00:00:00+00:00'
[2025-04-01T14:50:23.825+0000] {logging_mixin.py:190} INFO - Task instance is in running state
[2025-04-01T14:50:23.826+0000] {logging_mixin.py:190} INFO -  Previous state of the Task instance: queued
[2025-04-01T14:50:23.841+0000] {logging_mixin.py:190} INFO - Current task name:create_table state:running start_date:2025-04-01 14:50:21.786281+00:00
[2025-04-01T14:50:23.842+0000] {logging_mixin.py:190} INFO - Dag name:my_dag_name and current dag run status:running
[2025-04-01T14:50:23.844+0000] {taskinstance.py:732} INFO - ::endgroup::
[2025-04-01T14:50:23.935+0000] {connection.py:277} WARNING - Connection schemes (type: google_cloud_platform) shall not contain '_' according to RFC3986.
[2025-04-01T14:50:24.012+0000] {base.py:84} INFO - Retrieving connection 'gcp_co'
[2025-04-01T14:50:24.032+0000] {bigquery.py:1274} INFO - Creating table...
[2025-04-01T14:50:24.798+0000] {bigquery.py:1304} INFO - Table fake_news_detection.training_data already exists.
[2025-04-01T14:50:24.932+0000] {taskinstance.py:341} INFO - ::group::Post task execution logs
[2025-04-01T14:50:24.935+0000] {taskinstance.py:353} INFO - Marking task as SUCCESS. dag_id=my_dag_name, task_id=create_table, run_id=scheduled__2021-12-09T00:00:00+00:00, execution_date=20211209T000000, start_date=20250401T145021, end_date=20250401T145024
[2025-04-01T14:50:25.058+0000] {logging_mixin.py:190} INFO - Task instance in success state
[2025-04-01T14:50:25.062+0000] {logging_mixin.py:190} INFO -  Previous state of the Task instance: running
[2025-04-01T14:50:25.063+0000] {logging_mixin.py:190} INFO - Dag name:my_dag_name queued_at:2025-04-01 14:49:52.357768+00:00
[2025-04-01T14:50:25.064+0000] {logging_mixin.py:190} INFO - Task hostname:900d630d7d5f operator:BigQueryCreateTableOperator
[2025-04-01T14:50:25.104+0000] {local_task_job_runner.py:266} INFO - Task exited with return code 0
[2025-04-01T14:50:25.180+0000] {local_task_job_runner.py:245} INFO - ::endgroup::
